packages/haul-cli/src/commands/multiBundle.ts

```diff
@@ -0,0 +1,184 @@
+import { Arguments } from 'yargs';
+import webpack from 'webpack';
+import {
+  Runtime,
+  getProjectConfigPath,
+  getNormalizedProjectConfigBuilder,
+  NormalizedProjectConfig,
+} from '@haul-bundler/core';
+import * as messages from '../messages/multiBundleMessages';
+import SimpleProgressWebpackPlugin from 'simple-progress-webpack-plugin';
+
+export default function ramBundleCommand(runtime: Runtime) {
```

Comment:
Shouldn't it be called `multiBundleCommand`?

Reply:
fixed